src/unix/linux-core.c

```diff
@@ -90,7 +90,11 @@ int uv__platform_loop_init(uv_loop_t* loop) {
    * a.k.a. Lollipop. Since EPOLL_CLOEXEC is an alias for O_CLOEXEC on all
    * architectures, we just use that instead.
    */
+#if defined(__ANDROID_API__) && __ANDROID_API__ < 21
+  fd = errno = ENOSYS, -1;
```

Comment:
The `, -1` is a no-op here. I suspect you wanted to write this:
```c
fd = -1, errno = ENOSYS;
```
Ditto on line 296.

Reply:
Sorry, my mistake, fixed.